pkg/service/test/test.go

```diff
@@ -55,6 +55,20 @@ type TestOptions struct {
 	PassthroughHosts   []models.Filters
 }
 
+type Tuple struct {
+    First int
```

Comment:
tuple seems to be a generic name we want something for out context. Also rename first second to something with Keploy's. context like totaltest passed failed and report structure.

Reply:
done